test/project_types/theme/commands/generate/env_test.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+require 'project_types/theme/test_helper'
+
+module Theme
+  module Commands
+    module GenerateTests
+      class EnvTest < MiniTest::Test
+        resp = [200,
+                { "themes" =>
+                 [{ "id" => 2468,
+                    "name" => "my_theme" },
+                  { "id" => 1357,
+                    "name" => "your_theme" }] }]
```

Comment:
The indentation here is hurting my brain a bit, is this a rubocop thing or can we make it a little bit clearer?

Reply:
It is Rubocop; however, would it be better if more items were on the same line?

        resp = [200, { "themes" =>
                 [{ "id" => 2468, "name" => "my_theme" },
                  { "id" => 1357, "name" => "your_theme" }] }]